rosalind_hamm.py

```diff
@@ -0,0 +1,9 @@
+def main(data):
+    a, b = data[:-1].split('\n')
```

Comment:
Oh, I saw `[::-1]`, ugh, slice notation is kind of unreadable in its own way.

Ok, next time, use `data.strip()`, it'll take care of your trailing newline problem for you.